### DISCUSSION TITLE
[Q&A] 关于用户信息同步

### DISCUSSION THREAD

**Discussion by limfoo** [lobehub/lobehub#539] [General Question] [answered]
请教一下大家，如果用OneDrive同步"C:\Users\xxxx\AppData\Local\Microsoft\Edge\User Data\Default\IndexedDB\"路径下的.leveldb可以实现不同设备间的信息同步吗？或者还是需要像之前一样需要通过“导出-导入”的方式同步配置？谢谢。

**Comment by arvinxx** [comment, marked as answer]
跨端同步方案我们会做的，在 #368 跟进哈。

目前请使用导入导出功能。其他方式出现问题可能要自己解决

**Reply by limfoo** [reply]
收到，谢谢。
